extensions/sql-database-projects/src/models/project.ts

```diff
@@ -432,32 +436,41 @@ export class Project {
 		throw new Error(constants.unableToFindObject(path, constants.folderObject));
 	}
 
+	private addSystemDatabaseReferenceToProjFile(entry: SystemDatabaseReferenceProjectEntry): void {
+		const systemDbReferenceNode = this.projFileXmlDoc.createElement(constants.ArtifactReference);
+
+		// if it's a system database reference, we'll add an additional node with the SSDT location of the dacpac later
```

Comment:
you already added that below right? (later meaning below?)

Reply:
yup the later here means a few lines below